packages/babel-helper-environment-visitor/src/index.ts

```diff
@@ -0,0 +1,45 @@
+import type { NodePath } from "@babel/traverse";
+import { VISITOR_KEYS, staticBlock } from "@babel/types";
+import type * as t from "@babel/types";
+
+// TODO (Babel 8): Don't export this function.
+export function skipAllButComputedKey(
+  path: NodePath<t.Method | t.ClassProperty | t.ClassPrivateProperty>,
```

Comment:
It seems to me that `skipAllButComputedKey` is not invoked on `ClassPrivateProperty`.